src/installation/guides/chroot.md

```diff
@@ -0,0 +1,252 @@
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+NB For a UEFI booting system, make sure to create a UEFI System Partition (ESP).
+The ESP should have the partition type "EFI System" (code ef00) and be formatted
+as fat32 using [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+### Create a New Root and Mount Filesystems
+
+Create an empty directory to serve as a mount point for the new root filesystem,
+then mount any other filesystems on top of this new root. `/mnt` may be used
+instead of creating a new directory.
+
+This directory will be called `$VDROOT` for the remainder of this guide.
+
+```
+# VDROOT=/mnt
+```
+
+If using UEFI, mount the EFI System Partition as `$VDROOT/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `/dev/sda3` is to be
+booted as `/home` and `dev/sda1` is the EFI System Partition
+
+```
+# mkdir -p "$VDROOT"
+# mount /dev/sda2 "$VDROOT"/
+# mkdir "$VDROOT"/boot/
+# mount /dev/sda3 "$VDROOT"/home/
+# mkdir -p "$VDROOT"/boot/efi
+# mount /dev/sda1 "$VDROOT"/boot/efi
+```
+
+Initiliazile swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution (NB
+musl binaries are not available for i686).
+
+For example,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+xbps also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Three metapackages are available to boot strap Void: `base-system`,
+`base-voidstrap` and `base-minimal`. `base-minimal` is a strict subset of
+`base-voidstrap` and `base-voidstrap` is a strict subset of `base-system`. You
+may wish to compare these packages for yourself. `xbps-query -x [package]` will
+list dependencies pulled in for the given package. Even `base-system` is a very
+minimal installation.
+
+`base-system` is the package that is installed when using the interactive
+installer. `base-voidstrap` is the package that is installed in rootfs tarballs.
+
+`base-minimal` and `base-voidstrap` do not come with a linux kernel. The `linux`
+package always has the newest available kernel. Alternatively, a long term
+support kernel can be used by installing `linux-lts`.
+
+NB `base-minimal` does not include a
```

Comment:
I think we can stick to just recommending base-system for now. I don't want a ton of tangents in this guide. This would be another good topic to cover on a page about using different kernel versions maybe.